src/containers/ListingPage/listingSagas.js

```diff
@@ -21,6 +21,11 @@ export function* fetchListing() {
   if (!listings.results) {
     yield put(actions.setListing([]));
   } else {
+    const numberOfPages = Math.ceil(listings.totalCount / listings.pageSize);
+    for (let i = 2; i <= numberOfPages; i += 1) {
```

Comment:
Tenker man kunne gjort noe som `[2, 3, 4, ... N].forEach(x => ...)` i stedet for å bruke en for-loop